spec/system/referee_interface/referee_does_not_respond_spec.rb

```diff
@@ -16,63 +15,72 @@
     then_an_email_is_sent_to_the_candidate_asking_for_a_new_referee
 
     when_the_candidate_does_not_respond_within_28_days
-    then_the_candidate_is_sent_a_chase_email
-    and_the_referee_is_sent_a_chase_email
+    then_the_candidate_is_sent_a_final_chase_email
+    and_the_referee_is_sent_a_final_chase_email
   end
 
-  def given_a_candidate_completed_an_application
-    candidate_completes_application_form
-  end
-
-  def when_the_candidate_submits_the_application
-    candidate_submits_application
-    @application.application_references.first.update!(feedback_status: :feedback_refused)
+  def given_there_is_an_application_with_a_reference
+    @reference = create(:reference, :requested, email_address: 'anne@other.com', name: 'Anne Other')
+    @application = create(:application_form, first_name: 'F', last_name: 'B', application_references: [@reference])
   end
 
   def and_the_referee_does_not_respond_within_7_days
     Timecop.travel(7.days.from_now) do
-      SendReferenceChaseEmailToBothPartiesWorker.perform_async
+      ChaseReferences.perform_async
+      ChaseReferences.perform_async
```

Comment:
Is this to test only one email is sent? An explanatory comment might be useful here, to make it more obvious.